### DISCUSSION TITLE
Public Players Now PWAs

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4314] [Q&A]
Can someone please elaborate on where to find and how to implement this new feature from current version 0.13.0:

"The built-in public players are now Progressive Web Apps (PWAs) that can be "installed" on browsers and mobile devices."

**Comment by BusterNeece** [comment]
It means we've implemented all the necessary technical setup (application manifest file, service worker script, etc.) to allow browsers on both desktop and mobile devices to show a prompt similar to the one below when viewing our built-in public player page:

![2021-06-19 13_24_44-](https://user-images.githubusercontent.com/6744885/122651975-c9751c80-d101-11eb-8769-dca34fcd4c75.png)

**Reply by countrymusicfy** [reply]
That is great! Can you please tell me the location of the files to customize all PWA related items?

**Reply by countrymusicfy** [reply]
For instance, 

1. Which file do I edit to change the branding image and text on the splash page?
2. Which file do I edit to change the logo and text on the install app prompt?
3. How do I change the logo on the home screen button?

I am happy to modify the files via docker-compose.override.yml but not sure where to start.
